packages/projection/src/operators/Mappers/withCIP67.ts

```diff
@@ -3,25 +3,26 @@ import { ProducedUtxo, WithUtxo } from './withUtxo';
 import { isNotNil } from '@cardano-sdk/util';
 import { unifiedProjectorOperator } from '../utils';
 import groupBy from 'lodash/groupBy';
+import { Mint, WithMint } from './withMint';
 
 export interface CIP67Asset {
   assetId: Cardano.AssetId;
   policyId: Cardano.PolicyId;
   assetName: Cardano.AssetName;
   decoded: Asset.DecodedAssetName;
-  utxo: ProducedUtxo;
+  utxo?: ProducedUtxo;
 }
 
 export interface CIP67Assets {
-  byLabel: Partial<Record<Asset.AssetNameLabel, CIP67Asset[]>>;
-  byAssetId: Partial<Record<Cardano.AssetId, CIP67Asset>>;
+  byLabel: Partial<Record<Asset.AssetNameLabel, CIP67Asset[] | Mint[]>>;
+  byAssetId: Partial<Record<Cardano.AssetId, CIP67Asset | Mint>>;
```

Comment:
These doesn't seem right: output of this mapper should be CIP67 stuff
```suggestion
  byLabel: Partial<Record<Asset.AssetNameLabel, CIP67Asset[]>>;
  byAssetId: Partial<Record<Cardano.AssetId, CIP67Asset>>;
```

Reply:
addressed: 9a7e993dc1bcae2ac9d8b3df4b32cb1bac5e8371